p2panda-rs/src/schema/utils.rs

```diff
@@ -0,0 +1,32 @@
+/// All user schema
+pub const USER_SCHEMA: &str = r#"
+    userSchema = {
+        address // 
+        person
+    }
+    
+    address = (
+        city: { type: "str", value: tstr },
+        street: { type: "str", value: tstr },
+        house-number: { type: "int", value: int }, 
+    )
+    
+    person = (
+        name: { type: "str", value: tstr },
+        age: { type: "int", value: int }, 
+    )
+"#;
+
+/// All user schema hash
+pub const USER_SCHEMA_HASH: &str = "004069db5208a271c53de8a1b6220e6a4d7fcccd89e6c0c7e75c833e34dc68d932624f2ccf27513f42fb7d0e4390a99b225bad41ba14a6297537246dbe4e6ce150e8";
+
+/// Person schema
+pub const PERSON_SCHEMA: &str = r#"
+    person = (
+        name: { type: "str", value: tstr },
+        age: { type: "int", value: int }, 
+    )
+"#;
+
+/// Person schema hash
+pub const PERSON_SCHEMA_HASH: &str = "004069db5208a271c53de8a1b6220e6a4d7fcccd89e6c0c7e75c833e34dc68d932624f2ccf27513f42fb7d0e4390a99b225bad41ba14a6297537246dbe4e6ce150e8";
```

Comment:
This looks like testing data and not a particular protocol-level schema, can it be moved into the test modules?